index.js

```diff
@@ -26,6 +26,12 @@ var createSwaggerPage = (options) => {
         },
         apis: options.apis || []
     });
+	//Add any external definitions provided
+	for (var externalDefinition in options.definitions) {
+	    if(options.definitions.hasOwnProperty(externalDefinition)) {
```

Comment:
The line above is not relevant since you're looping on options.definitions, the property is always defined